src/PYTHON/fix_python_invoke.cpp

```diff
@@ -91,6 +94,8 @@ int FixPythonInvoke::setmask()
 
 void FixPythonInvoke::end_of_step()
 {
+  modify->clearstep_compute();
```

Comment:
Can you add a comment line or two to explain why you are wrapping the callback with clearstep/addstep?
I.e. in case the Python method accesses compute info that depends on pair energy/virial data being calculated on specific timesteps.   Ditto at the end of the constructor.

Reply:
And ditto in post_force().

Reply:
Added comments.